pkg/controllers/user/logging/utils/templatewrap.go

```diff
@@ -304,3 +342,81 @@ func parseEndpoint(endpoint string) (host string, scheme string, err error) {
 
 	return u.Host, u.Scheme, nil
 }
+
+func ValidateCustomTargetContent(content string, enableEscaped bool) (string, error) {
+	r := strings.NewReader(content)
+	buf := bufio.NewReader(r)
+	var newContent bytes.Buffer
+	for {
+		l, err := buf.ReadString('\n')
+		if l != "" {
+			line := strings.TrimSpace(l)
+			lineArray := strings.Split(line, " ")
+			if len(lineArray) == 0 {
+				continue
+			}
+
+			key := lineArray[0]
+			if err := filterFluentdTags(key); err != nil {
+				return "", err
+			}
+
+			if enableEscaped {
+				value := strings.TrimSpace(strings.TrimPrefix(line, key))
+				newValue := escapeString(value)
+				newContent.WriteString(key)
+				newContent.WriteString(" ")
+				newContent.WriteString(newValue)
+				newContent.WriteString("\n\t")
```

Comment:
Why `\t`?

Reply:
so the next line will not start at the beginning of a line. will like below 
```
<match>
   host xxx.com
</match>
```
not like this
```
<match>
host xxx.com
</match>
```

Reply:
We should put it before the `key`, right? it is quite confusing to add `\t` here.

Reply:
if we put it before the key the first line will like this:
```
<match>
         host xxx
    port 8088
    username name
</match>